tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/statements/transaction/TransactionStmtFlowTest.java

```diff
@@ -43,194 +42,145 @@ public void setup() {
     }
 
     @Test
-    public void testTransactionStmt() {
-        BValue[] returns = BRunUtil.invoke(programFile, "testTransactionStmt");
-
-        Assert.assertEquals(returns.length, 1);
-        Assert.assertEquals(returns[0].stringValue(), "start inTrx endTrx end");
-    }
-
-    @Test
-    public void testTransactionStmtAbort() {
-        BValue[] args = {new BInteger(0)};
-        BValue[] returns = BRunUtil.invoke(programFile, "testTransactionStmtAbort", args);
+    public void testTransactionStmt1() {
```

Comment:
Shall we add meaningful names for test methods rather than adding number to the end of the same method. Internally we can use the same ballerina function with the required arguments.